docs/application/native/guides/ui/efl/wearable/component-circle-rotary.md

```diff
@@ -80,8 +80,8 @@ To add an item:
 
    | Text                 | Setting function                         | View                                     |
    |--------------------|----------------------------------------|----------------------------------------|
-   | `selector,main_text` | `eext_rotary_selector_item_part_text_set()``eext_rotary_selector_item_domain_translatable_part_text_set()` | ![main_text](./media/rotary_selector_main_text.png) |
-   | `selector,sub_text`  | `eext_rotary_selector_item_part_text_set()``eext_rotary_selector_item_domain_translatable_part_text_set()` | ![sub_text](./media/rotary_selector_sub_text.png) |
+   | `selector,main_text` | `eext_rotary_selector_item_part_text_set()`<br> `eext_rotary_selector_item_domain_translatable_part_text_set()` | ![main_text](./media/rotary_selector_main_text.png) |
```

Comment:
Broken images in line 83 and lin3 84. It looks like images are in wrong folder. They are in `../media/` not in `./media/`.

Reply:
done